Makefile

```diff
@@ -222,4 +228,6 @@ fmt: format
 	fossa \
 	realtime-perftest \
 	format \
-	fmt
+	fmt \
+	lint \
+	$(NULL)
```

Comment:
@dankenigsberg Hi. Can you please elaborate on the usage of `$(NULL)` here?

Reply:
I can :)
The next time this list is updated, you will see only one line added/removed. `#(NULL)` is a trick to keep the tail stable.

These are one of the tricks that got lost with the old unix engineers moving away from code :)

Reply:
@EdDev Thank you!